lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -323,6 +327,9 @@ WasmBytecodeGenerator::GenerateFunction()
         {
             throw WasmCompilationException(_u("%s\n  Function %s"), ex.GetErrorMessage(), functionName);
         }
+
+        // Since we will continue compilation for other functions, clear the eval stack
+        m_evalStack.Clear();
```

Comment:
We create new register space for every functions, therefore, it is not needed to release the location of a function that failed.
We only need to clear that stack to make sure we have it fresh for the next function
I would like to move this stack to a more appropriate structure that would be per Function instead of on the ByteCodeGenerator itself. (will be in an effort to test deferred parse)